modules/machine-learning-services/workspaces/computes/metadata.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://aka.ms/bicep-registry-module-metadata-file-schema-v2#",
     "name": "Machine Learning Workspaces Computes",
-    "summary": "This module deploys computes for an Machine Learning Workspace.\n\nAttaching a compute is not idempotent and will fail in case you try to redeploy over an existing compute in AML (see parameter `deployCompute`).",
+    "summary": "This module deploys a Compute for a Machine Learning Workspace.\n\nAttaching a compute is not idempotent and will fail in case you try to redeploy over an existing compute in AML (see parameter `deployCompute`).",
     "owner": "Azure/module-maintainers"
-}
+}
```

Comment:
```suggestion
    "summary": "This module deploys a Machine Learning Services Workspaces Compute.\n\nAttaching a compute is not idempotent and will fail in case you try to redeploy over an existing compute in AML (see parameter `deployCompute`).",
    "owner": "Azure/module-maintainers"
}
```